### ISSUE TITLE
Dependencies aren't properly fetched

### ISSUE THREAD

**Issue by shrx** [pushrax/osxmpdkeys#1]
I installed `osxmpdkeys` with

```
sudo pip install pyobjc-core
sudo pip install osxmpdkeys
```

and then ran

```
mpdkeys
Traceback (most recent call last):
  File "/usr/local/bin/mpdkeys", line 3, in <module>
    from osxmpdkeys import Client
  File "/usr/local/lib/python2.7/site-packages/osxmpdkeys/__init__.py", line 1, in <module>
    from .client import *
  File "/usr/local/lib/python2.7/site-packages/osxmpdkeys/client.py", line 2, in <module>
    import osxmmkeys
ImportError: No module named osxmmkeys
```

**Comment by pushrax** [comment]
Odd, osxmmkeys should be pulled in automatically as a dependency. Does it work if you manually `pip install osxmmkeys`?

**Comment by shrx** [comment]
I installed `osxmmkeys` and then it complained:

```
mpdkeys
Traceback (most recent call last):
  File "/usr/local/bin/mpdkeys", line 3, in <module>
    from osxmpdkeys import Client
  File "/usr/local/lib/python2.7/site-packages/osxmpdkeys/__init__.py", line 1, in <module>
    from .client import *
  File "/usr/local/lib/python2.7/site-packages/osxmpdkeys/client.py", line 2, in <module>
    import osxmmkeys
  File "/usr/local/lib/python2.7/site-packages/osxmmkeys/__init__.py", line 1, in <module>
    from .tap import *
  File "/usr/local/lib/python2.7/site-packages/osxmmkeys/tap.py", line 1, in <module>
    import AppKit
```

So then I wanted to install `AppKit` but I haven't managed it yet because

```
sudo pip install Appkit
Collecting Appkit
  Downloading AppKit-0.2.8.tar.gz
Collecting flask (from Appkit)
  Downloading Flask-0.10.1.tar.gz (544kB)
    100% |################################| 544kB 326kB/s
Collecting pygobject (from Appkit)
  Downloading pygobject-2.28.3.tar.bz2 (889kB)
    100% |################################| 892kB 415kB/s
    ********************************************************************
    * Building PyGObject using distutils is only supported on windows. *
    * To build PyGObject in a supported way, read the INSTALL file.    *
    ********************************************************************
    Complete output from command python setup.py egg_info:
    ********************************************************************

    * Building PyGObject using distutils is only supported on windows. *

    * To build PyGObject in a supported way, read the INSTALL file.    *

    ********************************************************************

    ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in /private/tmp/pip-build-rWgR_o/pygobject
```

**Comment by pushrax** [comment]
Appkit comes from pyobjc, which is also listed as a dependency and should have been automatically installed...

Try `pip install pyobjc python-mpd2 argparse` and let me know how it goes.

**Comment by shrx** [comment]
Thanks, it works now.

**Comment by pushrax** [comment]
Going to reopen this for now since there is still an issue with the dependency fetching.

**Comment by pushrax** [comment]
Should be fixed in dcc90c57d192434d38b3502740f85a2f22a52eb4
